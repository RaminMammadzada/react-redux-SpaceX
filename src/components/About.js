import React, { Component } from 'react';

class About extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div>
        <p>About</p>
      </div>
    );
  }
}

export default About;
