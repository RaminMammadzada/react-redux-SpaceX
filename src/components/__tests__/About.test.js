import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../store';
import About from '../About';

describe('About component - ', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(
      <BrowserRouter>
        <Provider store={store}>
          <About />
        </Provider>
      </BrowserRouter>,
    );

    expect(component).toMatchSnapshot();
  });
});
