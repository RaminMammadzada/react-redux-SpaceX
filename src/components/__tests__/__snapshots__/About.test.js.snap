// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component -  matches the snapshot 1`] = `
<div
  className="m-5"
>
  <p>
    This is a capstone project for React and Redux curriculum in Microverse School. It is designed and implemented by Ramin Mammadzada.
  </p>
  <p>
    I am not affiliated, associated, authorized, endorsed by, or in any way officially connected with Space Exploration Technologies Inc (SpaceX), or any of its subsidiaries or its affiliates. The names SpaceX as well as related names, marks, emblems and images are registered trademarks of their respective owners.
  </p>
</div>
`;
